### DISCUSSION TITLE
macOS打包问题

### DISCUSSION THREAD

**Discussion by joriewong** [LeetaoGoooo/KindleHelper#26] [Q&A]
Catalina 10.15.5

**Comment by joriewong** [comment]
@lt94 麻烦空了debug下mac上的打包

**Comment by LeetaoGoooo** [comment]
你是在虚拟环境下打包的还是？
        默认python的环境是3.6的，如果不是在虚拟环境下的话，应该是 pip3 install -r requirements.txt 的，最后还需要将 kindleHelper.py 中有一行代码注释掉，不过macOS打包会有问题和aiohttp有关~~

**Comment by joriewong** [comment]
conda虚拟环境, Python 3.7

**Comment by 1-WEEK** [comment]
非虚拟环境，python3.7，无法打开构建出的应用

![image](https://user-images.githubusercontent.com/11807899/85984393-b3b3a680-ba1b-11ea-8aef-36284bacc730.png)

不熟悉 python 环境，能否直接上传编译好的版本，或者更新实际有效打包步骤，哪怕指明 kindleHelper.py 需要注释的代码也行。

**Comment by LeetaoGoooo** [comment]
macOS 为什么没有打包的原因是因为：

[pyinstaller cannot import aiohttp](https://github.com/aio-libs/aiohttp/issues/2301)

之前代码关于 macOS 涉及到 aiohttp 库的，我都用 requests 库重写了，这次新增的站点都没有改写~

依赖环境问题的多少是因为 anaconda 问题，我在 kindleHelper.py 中有一个代码专门在代码的时候时候使用的。
